BB-CEST/BB_RemotiveCloud

```diff
@@ -0,0 +1,57 @@
+Name 
+RemotiveCloud
+
+BB Tags(s) 
+BB-CEST; BB-CSC-TC
+
+Functional Cluster(s) (optional)	
+Diagnostics (could also be used for feeding vehicle data to emulators and other prototyping tools)
+
+Layer	
+Cloud
+             
+Known Implementation(s)	
+https://cloud.remotivelabs.com/ 
+
+ID (unique name)	
+
+Description	
+RemotiveCloud - for sharing vehicle recordings, both for debugging purposes as well as for prototyping (feed data into e.g. ProtoPie and Android AAOS / AOSP emulators). The recordings can be made with the RemotiveBroker or some other logging devices following standard formats.   
+Rationale	RemotiveCloud enables an easy way of sharing recordings - upload to cloud and share with relevant people as suppliers and service providers to have a common place for vehicle network data (CAN etc.) for debugging and 3rd party application developers.      
+
+Governance Applicable S-BB(s) 
```

Comment:
Are there any S-BBs under consideration like: EU General Data Protection Regulation (GDPR)?

Reply:
IMO, GPRD is not a building block but a large regulation that implies a number of specific functionalities (BBs) - these might become part of the FEDERATE catalog, but on the other hand, they are not at all automotive-specific, so I think they are out of SDV scope.

Reply:
For such Regulations we have defined the S-BB Tag, Supporting BBs which are regulations, standards ... 
If for example a vehicle recording (data set) is stored in the cloud and the vehicle owner can be identified on the basis of the stored data, ....